skaffold.yaml

```diff
@@ -7,32 +7,36 @@ build:
     useDockerCLI: true
     useBuildkit: true
   artifacts:
-  - image: gcr.io/k8s-staging-metrics-server/metrics-server
-    docker:
-      dockerfile: Dockerfile
-      buildArgs:
-        ARCH: "amd64"
-        GIT_TAG: "devel"
+    - image: gcr.io/k8s-staging-metrics-server/metrics-server
+      docker:
+        dockerfile: Dockerfile
+        buildArgs:
+          ARCH: "amd64"
+          GIT_TAG: "devel"
 deploy:
```

Comment:
We should modify it to the same format as the main branch
```

profiles:
  - name: test
    deploy:
      kustomize:
        paths:
          - manifests/test
  - name: test-ha
    deploy:
      kustomize:
        paths:
          - manifests/test-ha
```

Reply:
@yangjunmyfm192085 it looks like the block above this is the issue as the Kustomize deploy isn't scoped to a profile. I didn't want to change this previously as it wasn't clearly related to the Helm tests but it does explain how we've got multiple metrics-server installations when running the Helm tests.